Sources/FunctionalCss/Spacing.swift

```diff
@@ -105,7 +105,8 @@ private let spacings: [Size] = [
   .rem(0.5),
   .rem(1.0),
   .rem(2.0),
-  .rem(4.0)
+  .rem(4.0),
+  .rem(8.0)
```

Comment:
We need a new size of spacing for the hero area of the pricing page, since on desktop it has a healthy amount of padding (~128px)